packages/common/package.json

```diff
@@ -33,6 +33,7 @@
     "@ethersproject/bignumber": "^5.0.5",
     "@google-cloud/kms": "^0.3.0",
     "@google-cloud/storage": "^2.4.2",
+    "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@^0.3.0-beta.7",
```

Comment:
why this?

Reply:
This is effecetively installing `hardhat-deploy-ethers` via the recommended installation details below. This adds an `ethers` object to the `hardhat runtime environment` which provides useful utilities such as `getContractAt`.

```
if you use ethers.js we recommend you also install hardhat-deploy-ethers which add extra features to access deployments as ethers contract.

Since hardhat-deploy-ethers is a fork of @nomiclabs/hardhat-ethers and that other plugin might have an hardcoded dependency on @nomiclabs/hardhat-ethers the best way to install hardhat-deploy-ethers and ensure compatibility is the following:

npm install --save-dev  @nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers ethers
Which means you then need to do require("@nomiclabs/hardhat-ethers") instead of require("hardhat-deploy-ethers") in your hardhat.config.js file.

More details on hardhat-deploy-ethers repo : https://github.com/wighawag/hardhat-deploy-ethers#readme
```